Sources/RealModule/AugmentedArithmetic.swift

```diff
@@ -9,18 +9,85 @@
 //
 //===----------------------------------------------------------------------===//
 
+/// A namespace for "augmented arithmetic" operations for types conforming to
+/// `Real`.
+///
+/// Augmented arithmetic refers to a family of algorithms that represent
+/// the results of floating-point computations using multiple values such that
+/// either the error is minimized or the result is exact.
 public enum Augmented { }
 
 extension Augmented {
+  /// The product `a * b` represented as an implicit sum `head + tail`.
+  ///
+  /// `head` is the correctly rounded value of `a*b`. If no overflow or
+  /// underflow occurs, `tail` represents the rounding error incurred in
+  /// computing `head`, such that the exact product is the sum of `head`
+  /// and `tail` computed without rounding.
+  ///
+  /// This operation is sometimes called "twoProd" or "twoProduct".
+  ///
+  /// Edge Cases:
+  /// -
```

Comment:
There is a blank bullet here and also in some lists below. Is it intentional?

Reply:
Yes, this produces the desired formatting in Xcode: 
<img width="258" alt="Screen Shot 2021-08-19 at 2 21 14 PM" src="https://user-images.githubusercontent.com/641446/130123375-35797591-8dbb-44a6-ae97-9661579e0c01.png">